src/webportal/src/app/job-submission/components/tools/job-ssh.jsx

```diff
@@ -138,20 +154,33 @@ export const JobSSH = ({
           content={`Choose SSH public key for job. Users should maintain the SSH private key themselves.`}
         />
       </Stack>
-      <Hint>
-        Enable Users SSH to allow user attach job containers through
-        corresponding ssh private key.
-      </Hint>
       <Toggle
-        label='Enable Job SSH'
+        label={
+          <div>
+            Enable Job SSH{' '}
+            <TooltipHost content='Enable Job SSH to allow SSH between job containers through ssh <taskrole_name>-<taskrole_index>.'>
```

Comment:
How could user know the taskrole_index? As a user, I have no idea what taskrole_index should I use.

Reply:
User will not use this. Remove the details

Reply:
If user will not use this job ssh, why there is an option to let user select. Could it be enabled by default and removed from UI?

Reply:
The MPI template will use it.
This is a required feature for demo, we could refine the UE later.